components/GroupProfilePanels/GroupSettingsPanel.js

```diff
@@ -121,11 +121,11 @@ class GroupSettingsPanel extends Component {
       submitting,
     } = this.state
 
-    const address = changes.address || group.attributes.address
-    const description = changes.description || group.attributes.description
+    const address = typeof changes.address === 'string' ? changes.address : group.attributes.address
+    const description = typeof changes.description === 'string' ? changes.description : group.attributes.description
     // const discoverable = changes.discoverable || group.attributes.discoverable
```

Comment:
I'm not sure what type we should be expecting here. As such I've this one as is to be connected when the discoverable toggle is enabled.

Reply:
It should be a boolean, but we don't need to bother with it right now. 😉